3.cell_segmentation/scripts/1.segment_cells.py

```diff
@@ -29,62 +29,56 @@
     in_notebook = False
 
 
+# Get the current working directory
+cwd = pathlib.Path.cwd()
+
+if (cwd / ".git").is_dir():
+    root_dir = cwd
+
+else:
+    root_dir = None
+    for parent in cwd.parents:
+        if (parent / ".git").is_dir():
+            root_dir = parent
+            break
+
+# Check if a Git root directory was found
+if root_dir is None:
+    raise FileNotFoundError("No Git root directory found.")
+
+sys.path.append(f"{root_dir}/utils/")
+
+from parsable_args import parse_segmentation_args
```

Comment:
Is there a particular reason why the  import is required here?  If not, please consider moving it at the top of the notebook.